### ISSUE TITLE
Colorize not working with phi-3

### ISSUE THREAD

**Issue by ivanfioravanti** [ml-explore/mlx-examples#726]
I was testing logits probability with few models using --colorize and I noticed that phi-3 is not working

<img width="1728" alt="Colorize not working on phi-3 model" src="https://github.com/ml-explore/mlx-examples/assets/1069210/6be65f4e-678a-4fb6-a153-3f3976f42054">

**Comment by awni** [comment]
If it's still an issue, lmk and I will reopen.

**Comment by ivanfioravanti** [comment]
Thanks! it works!
